phsa-adapter/src/ViewModels/DmerInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Rsbc.Dmf.PhsaAdapter.ViewModels
+{
+    public class DmerFormInfo
```

Comment:
I wonder if this service should be aware of a DMER concept or just act as a proxy into another logical service to handle DMER specific future data